usr/local/sbin/restic_backup.sh

```diff
@@ -18,9 +18,9 @@ trap exit_hook INT TERM
 
 # Set up exclude files: global + path-specific ones.
 exclusion_args="--exclude-file /etc/restic/backup_exclude"
-for homedir in /home/*; do
-	if [ -f "$homedir/.backup_exclude" ]; then
-		exclusion_args+=" --exclude-file $homedir/.backup_exclude"
+for backup_path in ${BACKUP_PATHS[@]}; do
```

Comment:
Question here: What will happen if `BACKUP_PATHS=/home/` and, then there is a `/home/someuser/.backup_exclude` file? The file makes reference to a specific user, while the paths refer to the higher level directory `/home`.

The previous behaviour takes the `someuser` exclusion file in consideration, but after the change what will happen? I think it will be omitted? If that's right, may be confusing. Did I get it right? What do you think? :D

Maybe aside of `backup_paths/.backup_exclude`, also all user homedir `.backup_exclude` can be considered.

Reply:
Ah this was the original intention, got-cha! Thanks

I had forgotten. We should look for `.backup_exclude` for each mount-point, and additionally each user dir in `/home`, if `/home` is in the backup path.

Will fix!

Reply:
Cool 😊

Reply:
So, please check 424bb0c117285de5a3a22aee35f37544ec4a6fcf

I tried it out locally, seemed to work... 😅